architecture/Threatmodel.md

```diff
@@ -0,0 +1,128 @@
+### Non Repudiation 
+Niet van Toepassing 
+###  Information Disclosure
+Hierbij kunnen andere clients dan de officiele app ook de Temporary exposure keys downloaden.  Om dit te vermijden dienen er maatregelen genomen worden zodat de informatie alleen beschikbaar is voor de app. 
+Ook dient de verbinding tussen de App en de centrale server beveiligd te zijn, zodat deze niet ontsleuteld kan worden door tussenliggende partijen. TLS met daarbij Certificate pinning wordt aangeraden. 
+De server dient ook beschermt te zijn tegen ongeauthorizeerde toegang via andere kanalen dan de download faciliteit die de app gebruikt. De gebruikelijke OS permissie, OS hardening, software update en authenticatie maatregelen dienen genomen te worden. 
+### Denial of Service
+Omdat elke App de bestanden download kan men eenvoudig de adressen van de servers achterhalen door de netwerk activiteit van de app te observeren. Dus het is een risico dat deze servers blootgesteld worden aan Denial of Service Attacks. 
+### Elevation of Privilege
+Niet van toepassing privileges/authenticatie worden in dit proces zoals nu bekend niet gebruikt.
+
+## 3. Het uitvoeren van een test en het doorgeven van een positieve test uitslag.
+### Spoofing
+De test en uitslagen worden verwerkt via een web applicatie. TLS encryptie dient te worden uitgevoerd. De OWASP richtlijnen voor veilige web applicaties, om risico's zoals sql injection, cross site scripting en dergelijke uit te sluiten.   
+### Tampering
+Toegang tot de databases waarin tests worden opgelagen dient goed te worden geregeld. Daarnaast dient het systeem van de TAN code's ook cryptografisch geanalyseerd te worden. 
+### Non Repudiation 
+Gebruikers kunnen een behoorlijk belang hebben bij een negatieve test uitslag omdat een postieve verplicht tot quarantaine. Indien er feedback 
+### Information Disclosure
+Informatie over de test uitslag is privacy gevoelig en dient te worden afgeschermd. 
+### Denial of Service
+Het doen van de test en dan het weer doorgeven van een positieve test gebruiken ook een webapplicatie. Een denial of serive attack zou hier mogelijk kunnen zijn. 
+### Elevation of Privilege 
+Het stuk waarin informatie wordt opgeslagen dient gebruikers te authoriseren en kent mogelijk meerde privileges. Ook een functionaliteit om mogelijk een fout-positieve of fout-negatieve test later weer te corrigeren is een stuk van de applicatie waarin risico's worden gelopen. Er kan druk worden uitgeoefend om test resultaten later onterecht van uitslag te veranderen.  
+## 4. Installatie en onboarding Risco's 
+###  Information Disclosure
+Informatie over de installatie van uit de App Store/ PlayStore is bekend bij Apple/Google en statistieken zijn hierover te verkrijgen.
+### Non Repudiation 
+Een druk om de app te installeren in bepaalde iets meer riskante bijeenkomsten (denk aan horeca, sport of OV)  is te verwachten.  
+  ### Denial of Service
```

Comment:
spaties zorgen ervoor dat dit niet in alle markdown renderers wordt gezien als kopje